mifosng-android/src/main/java/com/mifos/mifosxdroid/online/sharingaccount/SharingAccountFragment.java

```diff
@@ -0,0 +1,421 @@
+
+    @BindView(R.id.sp_charges)
+    Spinner spCharges;
+
+    @BindView(R.id.sp_lockin_period)
+    Spinner spLockinPeriod;
+
+    @BindView(R.id.sp_min_active_period)
+    Spinner spMinActivePeriod;
+
+    @BindView(R.id.sp_savings_account)
+    Spinner spSavingsAccount;
+
+    @BindView(R.id.et_client_external_id)
+    EditText etClientExternalId;
+
+    @BindView(R.id.tv_submittedon_date)
+    TextView tvSubmissionDate;
+
+    @BindView(R.id.tv_application_date)
+    TextView tvApplicationDate;
+
+    @BindView(R.id.et_requested_shares)
+    EditText etRequestedShares;
+
+    @BindView(R.id.et_lockin_period)
+    EditText etLockinPeriod;
+
+    @BindView(R.id.et_min_active_period)
+    EditText etMinActivePeriod;
+
+    @BindView(R.id.cb_allow_dividend_calculation)
+    CheckBox cbDividendAllowed;
+
+    @BindView(R.id.recycler_charges)
+    RecyclerView recyclerSheets;
+
+    @BindView(R.id.btn_submit)
+    Button btnSubmit;
+
+    @BindView(R.id.img_add)
+    ImageView btnAdd;
+
+    @Inject
+    SharingAccountPresenter mSharingAccountPresenter;
+
+    @Inject
+    ShareChargesAdapter shareChargesAdapter;
+
+    private View rootView;
+
+    private DialogFragment mfDatePicker;
+    private int productId;
+    private int clientId;
+    private int savingId;
+    private int lockinPeriodId;
+    private int minActivePeriodId;
+    private int chargeId;
+    private int defaultShares;
+    private String submissionDate;
+    private String applicationDate;
+    private boolean isAllowDividend = false;
+    private boolean isApplicationDate = false;
+    private boolean isSubmissionDate = false;
+
+    List<String> mChargesNames = new ArrayList<>();
+    List<String> mListSharingProductsNames = new ArrayList<>();
+    List<String> mListLockinPeriodOptions = new ArrayList<>();
+    List<String> mListMinActivePeriodOptions = new ArrayList<>();
+    List<String> mListSavingsAccountOptions = new ArrayList<>();
+
+    ArrayAdapter<String> mChargesAdapter;
+    ArrayAdapter<String> mSharingProductsAdapter;
+    ArrayAdapter<String> mLockinPeriodAdapter;
+    ArrayAdapter<String> mMinActivePeriodAdapter;
+    ArrayAdapter<String> mSavingsAccountAdapter;
+
+    private SharingAccountsTemplate mSharingProductsTemplateByProductId;
+    private List<ProductSharing> mProductSharings;
+    private List<ChargeOptions> mChargeOptions;
+    private ChargeOptions chargeOptionSelected;
+
+    public static SharingAccountFragment newInstance(int clientId) {
+        SharingAccountFragment sharingAccountFragment = new SharingAccountFragment();
+        Bundle args = new Bundle();
+        args.putInt(Constants.CLIENT_ID, clientId);
+        sharingAccountFragment.setArguments(args);
+        return sharingAccountFragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
+        if (getArguments() != null)
```

Comment:
Use braces to enclose the `if` statement

Reply:
done.